### PR TITLE
Feat/aag 1967 the main list becomes unscrollable

### PR DESCRIPTION
**Pull request by thomas-james-1986**
This PR fixes the main screen of the app becoming unscrollable after an error alert is displayed on a placement detail page. 
This was caused by the `SAAlert` class which seemed a bit overly complex so I've replaced it with a new `Alert` type.

This PR also adds an initial `config.yml` that runs SwiftLint using the PopJam ruleset to fix the CI breaking.

### CODE REVIEW

File: AwesomeAdsDemo.xcworkspace/xcuserdata/tom.orourke.xcuserdatad/xcdebugger/Breakpoints_v2.xcbkptlist [hunk 1]
```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (tashmulla) [inline AwesomeAdsDemo.xcworkspace/xcuserdata/tom.orourke.xcuserdatad/xcdebugger/Breakpoints_v2.xcbkptlist:1]: Are these user data files needed or can we stick them into a `.gitignore`? 🙂

File: Podfile [hunk 2]
```diff
@@ -13,7 +13,6 @@ target 'AwesomeAdsDemo' do
   pod 'RxCocoa', '5.1.1'
   pod 'SuperAwesome', '7.2.18' #'7.2.5'
```
Comment (tashmulla) [inline Podfile:14]: Oooh I wonder if bumping up our SA Pod version will help with loading the ads properly? Something to look into separately I guess 👌

File: .swiftlint.yml [hunk 3]
```diff
@@ -0,0 +1,245 @@
...
+    message: "Fix before merging"
+    severity: error
+
+  function_whitespace:
+    name: "Function Whitespace"
+    regex: "([ ]*)([f][u][n][c]{1})([ ]{1})([ a-zA-Z0-9_.-]*)([ ]+)([(]{1})" # matches '// WARNING' (case insensitive)
+    match_kinds:
+      - identifier
+      - keyword
+    message: "Remove the space before the parnetheses"
```
Comment (tashmulla) [inline .swiftlint.yml:19]: Small typo - parentheses 😄

**Review by tashmulla — APPROVED** [review]
Nice! Just a comment about files potentially for a gitignore but otherwise all good!

File: .swiftlint.yml (hunk 3 above)
Comment (thomas-james-1986) [reply .swiftlint.yml:19]: Nice spot!

File: AwesomeAdsDemo.xcworkspace/xcuserdata/tom.orourke.xcuserdatad/xcdebugger/Breakpoints_v2.xcbkptlist (hunk 1 above)
Comment (thomas-james-1986) [reply AwesomeAdsDemo.xcworkspace/xcuserdata/tom.orourke.xcuserdatad/xcdebugger/Breakpoints_v2.xcbkptlist:1]: Good idea, they're mostly noise I think

File: Podfile (hunk 2 above)
Comment (thomas-james-1986) [reply Podfile:14]: Ah good hint, I'll look into it

**Pull request merged**